modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsRegistryImpl.java

```diff
@@ -42,10 +43,18 @@ public List<ModelPreFilterContributor> filterModelPreFilterContributor(
 			_retainAll(modelPreFilterContributors, _getMandatoryContributors());
 		}
 		else {
+			if ((excludes != null) && (excludes.size() == 1) &&
```

Comment:
Does this mean, that in case excludes = "*", even the mandatory contributor classes below are ignored?

Reply:
No we shouldn't, I've fixed the logic.